cfgov/v1/tests/test_author_names.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from v1.models.base import CFGOVPage
+
+class TestAuthorNames(TestCase):
+
+    def test_names_get_sorted_alphabetically(self):
+        page = CFGOVPage()
+        page.authors.add('Ross Karchner','Richa Agarwal', 'Andy Chosak', 'Will Barton')
+        expected_result = ['Richa Agarwal', 'Will Barton', 'Andy Chosak', 'Ross Karchner']
+        self.assertEquals(page.get_authors(), expected_result)
+    
+    def test_no_authors(self):
+        page = CFGOVPage()
+        self.assertEquals(page.alphabetize_authors(), [])
```

Comment:
i understand that get_authors() calls alphabetize_authors() anyway, but shouldn't this call the get_authors() wrapper like you did on line 10?  that way if you change the implementation of how you alphabetize things, you won't have to change this line in your tests.  same on line 20.